manifests/cf/cell.yml

```diff
@@ -85,3 +90,6 @@ instance_groups:
 
     - name: silk-cni
       release: silk
+      properties:
+        dns_servers:
+          - 169.254.0.2
```

Comment:
I'm assuming Garden and/or Silk does some magic to make the cell-local 169.x.x.x APIPA address available to the network namespace of each container so that they can resolve through the local DNS resolver and out?